types/tailwindcss/tailwindcss-tests.ts

```diff
@@ -2551,4 +2568,61 @@ colors.blueGray[500];
 
 // @ts-expect-error
 defaultTheme.darkMode;
+
+// @ts-expect-error
```

Comment:
Why would doing `defaultTheme.colors.blue[800];` cause a `ts-error`?

Reply:
`colors` is possibly `undefined`.
In this PR the `theme` now has `Partial` and all of its sub props are optional. This includes the `colors` prop and it doesn't explicitly need to be declared

Reply:
I should probably provide explanatory comments besides the `ts` directives so that it would give you an idea of what's expected from that error. Sorry about that 😓